packages/ui/src/design-system/Select/components/Options.tsx

```
@@ -210,7 +252,10 @@ const Options = ({
                           renderOptionItem={renderOptionItem}
                           createProps={createProps}
                           uppercase={uppercase}
-                          isKeyboardHighlighted={flatIdx === highlightedIndex}
+                          isKeyboardHighlighted={
+                            flatIdx >= 0 && flatIdx === highlightedIndex
+                          }
+                          onFocusOption={() => setHighlight(flatIdx)}
```

ESLint

[error] Error: Cannot access refs during render

React refs are values that are not needed for rendering. Refs should only be accessed outside of render, such as in event handlers or effects. Accessing a ref value (the `current` property) during render can cause your component not to update as expected (https://react.dev/reference/react/useRef).

/home/jailuser/git/packages/ui/src/design-system/Select/components/Options.tsx:258:48
  256 |                             flatIdx >= 0 && flatIdx === highlightedIndex
  257 |                           }
> 258 |                           onFocusOption={() => setHighlight(flatIdx)}
      |                                                ^^^^^^^^^^^^ Passing a ref to a function may read its value during render
  259 |                         />
  260 |                       );
  261 |                     })}

(react-hooks/refs)